src/views/Home.vue

```diff
@@ -147,21 +148,39 @@ export default {
         this.validators.blank.name = this.$validateIsBlank(value.url)
       },
       deep: true
+    },
+    authentication_token: function (value) {
+      if (value !== '') {
+        this.loadResources()
+      }
     }
   },
+  created () {
+    this.login({
+      username: 'gorums',

```

Comment:
can we remove those credentials and move them to a .env file for now until we have the login page??